controls/SV-235154.rb

```diff
@@ -64,12 +64,24 @@
 
   ssl_params = sql_session.query(query_ssl_params).results
 
-  describe '@@ssl_fips_mode' do
-    subject { ssl_params.column('@@ssl_fips_mode').join }
-    it { should match /ON|STRICT/ }
-  end
-  describe '@@require_secure_transport' do
-    subject { ssl_params.column('@@require_secure_transport').join }
-    it { should match /1|ON/ }
-  end
+  if !input('aws_rds')
+
+    describe '@@ssl_fips_mode' do
+      subject { ssl_params.column('@@ssl_fips_mode').join }
+      it { should match /ON|STRICT/ }
+    end
+    describe '@@require_secure_transport' do
+      subject { ssl_params.column('@@require_secure_transport').join }
+      it { should match /1|ON/ }
+    end
+    
+  else
+    
+    impact 0.0
+    describe 'Not applicable since ssl_fips_mode is set to 0 (OFF) and cannot be configured in AWS RDS' do
```

Comment:
No, because the reason why this is Not Applicable is only because of ssl_fips_mode and if we were to give a pass to require_secure_transport then folks might see the pass and assume FIPS mode is compliant when in fact it is not.

Reply:
Should the describe block for `require_secure_transport` be moved out of the condition since it's applicable to AWS RDS?